### DISCUSSION TITLE
npm | Node.js с примерами кода

### DISCUSSION THREAD

**Discussion by giscus** [xsltdev/nodejsdev.ru#30] [Announcements]
# npm | Node.js с примерами кода

Справочник web-разработчика с примерами кода - Node.js

https://nodejsdev.ru/doc/npm/

**Comment by Typodin** [comment]
"Если при установке пакета вы не хотите, чтобы он добавлялся в список зависимостей, то просто не указывайте флаг --save." (с) он добавится и без флага в список зависимостей dependencies, так как отсутсвие флага - это как раз --save по-умолчанию.

**Reply by bndby** [reply]
Да, теперь поведение именно такое, как вы описали, раньше нужно было явно --save писать.
